CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ This beta release is built on top of [OpenTelemetry .NET](https://github.com/ope
  `ConfigureTracesOptions(StackExchangeRedisCallsInstrumentationOptions options)`.
 - Add plugin support for
  `ConfigureMetricsOptions(AspNetCoreMetricsInstrumentationOptions options)`.
+- Add automatic assembly redirection for .NET Framework applications, see
+ `OTEL_DOTNET_AUTO_NETFX_ASSEMBLY_REDIRECTION_ENABLED`.
```

Comment:
If you are using, _see_. It will be great to have link to documentation. You can change it to something like

_use_. or _It can be configured by_